docs/01-get-started/your-first-route.mdx

```diff
@@ -6,12 +6,48 @@ sidebar_position: 3
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+# Routes
 
-Add instructions here:
+Starting v3.0.0, Front-Commerce uses Remix's
+[API Routes](https://remix.run/docs/en/main/guides/api-routes) to handle all the
+routing of you application. All routes are located under `app/routes` folder.
 
-- Create a route
-- Introduce loader
-- Redirect to Remix guides
+API Routes will use the default export of each Route file to generate the
+route's UI. Go ahead and create a new route `hello-world.tsx`:
 
-:::
+```tsx title="app/routes/hello-world.tsx"
+export default function HelloWorld() {
+  return (
+    <div>
+      <h1>Hello world!</h1>
+    </div>
+  );
+}
+```
+
+# Loaders
+
+In order to display dynamic content in your routes, you will need to fetch some
+data. This is done by using Remix's Routes API method `loader`.
+
+Here is a basic example of how a `loader` could be used:
+
+```tsx title="app/routes/hello-world.tsx"
+export const loader = () => {
+  // This method will be called prior to rendering to fetch the data for your component.
+  return { name: "Jack Doe" };
+};
+
+export default function HelloWorld() {
+  const { name } = useLoaderData();
+
+  return (
+    <div>
+      <h1>Hello {name}!</h1>
+    </div>
+  );
+}
+```
+
```

Comment:
**thought:** there are many other APIs for routes in Remix. I think it'd be great to have a "kitchen sink" with additional features to act a bit as a reference for developers.

This way, they'll understand how **central** routes are in a Remix (and FC) app.

**suggestion:** add a section "And many more features".
> Routes play a fundamental part in a Remix (and Front-Commerce) application. It is the glue between the URL requested by a user and the response your application will send.
> We will continue to learn how to use them for the most common actions in the next sections, but here is a preview of everything a route could do
> **Example of the Hello world page with meta tags, fake link resources etc…** 

Prompt for https://chat-remixdocs.vercel.app/ to kickstart this section
> Create a kitchen sink for a route, with all features, so a newbie can understand every responsibilities a route has

Reply:
Added a "full" example, based on what we already had and what Remix docs provides as examples.